### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by chrysalag**
## Story
https://trello.com/c/S2hGGyeM/16-as-a-developer-i-want-to-add-missing-translations

## Goals
- [x] Story is tested
- [x] Acceptance criterias are met
- [x] Code is reviewed
- [x] Travis is green
- [x] Story is on resolved in trello
- [x] Readme is updated if needed
- [x] Styled with Bootstrap v3
- [x] Mobile view is usable
- [x] Seeds are available
- [x] German translations available

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -154,29 +201,72 @@ de:
         mailer:
           invitation_instructions:
             accept_until_format: "%d. %B %Y %H:%M"
+  gender:
+    female: Weiblich
+    male: Männlich
+  language_level:
+    native_speaker: Muttersprachler
+    fluent: Fließend
```
Comment (casaper) [inline config/locales/de.yml]: Fliessend -> swiss speciality. We do not use the ß in Switzerland.

File: config/locales/de.yml [hunk 2]
```diff
@@ -154,29 +201,72 @@ de:
...
+    good: Gut
+    basic: Elementar
```
Comment (casaper) [inline config/locales/de.yml]: Grundlagen

File: config/locales/de.yml [hunk 3]
```diff
@@ -154,29 +201,72 @@ de:
...
+  role:
+    superadmin: Superadmin
+    admin: Admin
+    social_worker: Sozialarbeiter
+    department_manager: Standortverantwortlich
```
Comment (casaper) [inline config/locales/de.yml]: Standortverantwortliche/er

File: config/locales/de.yml [hunk 4]
```diff
@@ -154,29 +201,72 @@ de:
...
+    male: Männlich
+  language_level:
+    native_speaker: Muttersprachler
+    fluent: Fließend
+    good: Gut
+    basic: Elementar
+  role:
+    superadmin: Superadmin
+    admin: Admin
+    social_worker: Sozialarbeiter
```
Comment (casaper) [inline config/locales/de.yml]: SozialarbeiterIn

File: config/locales/de.yml [hunk 5]
```diff
@@ -154,29 +201,72 @@ de:
...
+    saturday: Samstag
+    sunday: Sonntag
+  add_language: Sprache hinzufügen
+  add_relative: Verwandte hinzufügen
+  are_you_sure: Sind Sie sicher?
+  are_you_sure_invitation: Sind Sie sicher, dass Sie eine Einladung verschicken möchten?
+  back: Zurück
   change_my_password: Ihr Passwort ändern
   change_password: Ihr Passwort ändern
+  client_created: Klient wurde erfolgreich erstellt.
```
Comment (casaper) [inline config/locales/de.yml]: KlientIn

File: config/locales/de.yml [hunk 6]
```diff
@@ -154,29 +201,72 @@ de:
...
+  client_updated: Klient wurde erfolgreich aktualisiert.
```
Comment (casaper) [inline config/locales/de.yml]: KlientIn

File: config/locales/de.yml [hunk 7]
```diff
@@ -154,29 +201,72 @@ de:
...
+  client_destroyed: Klient wurde erfolgreich gelöscht.
```
Comment (casaper) [inline config/locales/de.yml]: KlientIn

File: config/locales/de.yml [hunk 8]
```diff
@@ -154,29 +201,72 @@ de:
...
+  clients: Klienten
+  contact_info: Kontakt Informationen
+  create_client: Klient erfassen
```
Comment (casaper) [inline config/locales/de.yml]: KlientIn

File: config/locales/de.yml [hunk 9]
```diff
@@ -154,29 +201,72 @@ de:
...
   forgot_password: Passwort vergessen?
   invite_sent: 'Einladung wurde an %{email} verschickt.'
+  language_skills: Sprachkenntnisse
   login: Anmelden
   logout: Abmelden
   password_reset_instructions: Schicken Sie mir Passwortrücksetzung Instruktionen
+  personal_background: Persönlicher Hintergrund
   profile_created: Profil wurde erfolgreich erstellt.
   profile_updated: Profil wurde erfolgreich aktualisiert.
+  registered_by: 'Zuständige/r Sozialberater/in'
```
Comment (casaper) [inline config/locales/de.yml]: SozialberaterIn

File: config/locales/de.yml (hunk 4 above)
Comment (chrysalag) [inline config/locales/de.yml]: In future kids they use that form: Sozialarbeiter/in. 
I don't know which is more correct, so I'll trust you and @nuritnt on which one to use.

File: app/views/clients/_relatives_show.html.slim [hunk 10]
```diff
@@ -4,7 +4,7 @@
       = rel.first_name
       = ' '
       = rel.last_name
-      = ', '
```
Comment (nuritnt) [inline app/views/clients/_relatives_show.html.slim]: Why did you remove the colon?
The colon was there, to make the difference sure between firstname and lastname.
Because there are always confusions if there are two firstname or two lastnames.

Comment (chrysalag) [inline app/views/clients/_relatives_show.html.slim]: Because one didn't fill out all fields, multimple commas where shown.

File: config/locales/de.yml [hunk 11]
```diff
@@ -154,29 +201,72 @@ de:
...
   remember_me: Angemeldet bleiben
+  remove_language: Sprache entfernen
+  remove_relative: Verwandte entfernen
+  schedule: Zeitplan
   show_my_profile: Mein Profil
   show_profile: Profil anzeigen
+  when: Zeit
   users: Benutzer
-  user_index: Benutzer
   voluntary_platform: Freiwilligen Platform
```
Comment (nuritnt) [inline config/locales/de.yml]: Plattform

File: config/locales/de.yml [hunk 12]
```diff
@@ -154,29 +201,72 @@ de:
...
   create_user: Benutzer erfassen
   create_profile: Profil erfassen
   edit_login: Login bearbeiten
   edit_profile: Profil bearbeiten
   forgot_password: Passwort vergessen?
   invite_sent: 'Einladung wurde an %{email} verschickt.'
+  language_skills: Sprachkenntnisse
   login: Anmelden
   logout: Abmelden
   password_reset_instructions: Schicken Sie mir Passwortrücksetzung Instruktionen
```
Comment (nuritnt) [inline config/locales/de.yml]: Suggestion: "Ich möchte mein Passwort zurücksetzen."

File: config/locales/en.yml [hunk 13]
```diff
@@ -104,35 +105,76 @@ en:
...
   create_user: Create user
   create_profile: Create profile
   edit_login: Edit login
   edit_profile: Edit profile
   forgot_password: Forgot your password?
+  language_skills: Language skills
   login: Log in
   logout: Logout
   invite_sent: 'Invitation sent to %{email}'
   password_reset_instructions: Send me password reset instructions
```
Comment (nuritnt) [inline config/locales/en.yml]: Suggestion: "I want to reset my password."

**Comment by chrysalag** [comment]
@casaper @nuritnt Is there anything else that should be changed?
